{"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"role":"symbol","title":"ZendeskEvent.authenticationFailed(error:)","symbolKind":"case","modules":[{"name":"ZendeskSDK"}],"roleHeading":"Case","externalID":"s:10ZendeskSDK0A5EventO20authenticationFailedyACs5Error_p_tcACmF"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"]}],"schemaVersion":{"patch":0,"minor":3,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authenticationFailed"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}]},{"kind":"parameters","parameters":[{"name":"error","content":[{"inlineContent":[{"text":"The error that caused the authentication failure.","type":"text"}],"type":"paragraph"}]}]}],"hierarchy":{"paths":[["doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"]]},"identifier":{"url":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","interfaceLanguage":"swift"},"sections":[],"abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"kind":"symbol","references":{"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent/authenticationFailed(error:)":{"type":"topic","url":"\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)","title":"ZendeskEvent.authenticationFailed(error:)","kind":"symbol","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","abstract":[{"text":"Invoked when a rest call fails for authentication reasons.","type":"text"}],"role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"authenticationFailed","kind":"identifier"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":")","kind":"text"}]},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK":{"type":"topic","role":"collection","kind":"symbol","title":"ZendeskSDK","identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","url":"\/documentation\/zendesksdk","abstract":[]},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent":{"title":"ZendeskEvent","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"ZendeskEvent","kind":"identifier"}],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent","kind":"symbol","role":"symbol","abstract":[{"type":"text","text":"The set of events that can be emitted from Zendesk SDK."}],"navigatorTitle":[{"text":"ZendeskEvent","kind":"identifier"}],"url":"\/documentation\/zendesksdk\/zendeskevent","type":"topic"}}}